File: 920-uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.java
class Solution {
    public String[] uncommonFromSentences(String s1, String s2) {
        String[] arr1 = s1.split(" ");
        String[] arr2 = s2.split(" ");
        HashMap<String, Integer> hm = new HashMap<>();
        for (String st : arr1) {
            hm.put(st, hm.getOrDefault(st, 0) + 1);
        }
        for (String st : arr2) {
            hm.put(st, hm.getOrDefault(st, 0) + 1);
        }
        List<String> result = new ArrayList<>();
        for (Map.Entry<String, Integer> entry : hm.entrySet()) {
            String key = entry.getKey();
            int value = entry.getValue();
            if (value == 1) {
                result.add(key);
            }
        }
        return result.toArray(new String[0]);
    }
}

File: 920-uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.py
class Solution:
    def uncommonFromSentences(self, s1: str, s2: str) -> List[str]:
        s1 = s1.split(' ')
        s2 = s2.split(' ')
        dic = {}
        ans = []
        for i in s1:
            if i in dic:
                dic[i]+=1
            else:
                dic[i] = 1
        for i in s2:
            if i in dic:
                dic[i]+=1
            else:
                dic[i] = 1
        for k,v in dic.items():
            if v == 1:
                ans.append(k)
        return ans



        